src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -199,11 +199,24 @@ class JesBackend extends Backend with LazyLogging {
       case (fileTag: String, location: WdlFile) =>
         logger.info(s"${makeTag(backendCall)}: $fileTag -> ${location.valueString}")
         Seq(mkInput(fileTag, location))
-      case (fileTag: String, location: WdlArray) => location.value.collect { case f: WdlFile => mkInput(fileTag, f) }
+      case (fileTag: String, location: WdlArray) => inputsFromArray(fileTag, location.value, backendCall)
```

Comment:
scottfrazer added a note 29 minutes ago
Why fileTag and not fqn?
 @scottfrazer   Owner
scottfrazer added a note 29 minutes ago
Also maybe array: WdlArray instead of location: WdlArray
 @scottfrazer   Owner
scottfrazer added a note 11 minutes ago
as discussed in person, fileTag really is a locallyQualifiedName